pkg/agent/openflow/multicluster.go

```diff
@@ -82,6 +82,7 @@ func (f *featureMulticluster) l3FwdFlowToRemoteViaTun(
 			Cookie(cookieID).
 			MatchProtocol(ipProtocol).
 			MatchDstIPNet(peerServiceCIDR).
+			Action().LoadPktMarkRange(0x4d2, multiclusterPktMarkRange).
```

Comment:
multiclusterPktMark?
You may define `multiclusterPktMark` as `MulticlusterPktMark` in openflow package and use it in mc_route_controller.go